.github/workflows/ci.yml

```diff
@@ -15,10 +15,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - name: Set up Python 3.11
+      - name: Set up Python 3.12
         uses: actions/setup-python@v3
         with:
-          python-version: '3.11'
+          python-version: '3.12'
       - uses: actions/setup-node@v3
```

Comment:
This might be an excellent opportunity to upgrade the versions of the actions we use ;)